python/lsst/phosim/utils/phosim_repackager.py

```diff
@@ -105,6 +105,7 @@ def __init__(self, instName="lsst"):
         """
         # Use appropriate obs_lsst mapper camera object
         # and telescope code
+        self.image_type = image_type
```

Comment:
Please consider add the comment to explain the meaning of this new attribute.

Reply:
Added a  comment